support-frontend/assets/components/directDebit/directDebitProgressiveDisclosure/components/form.jsx

```diff
@@ -90,29 +94,32 @@ function Form(props: PropTypes) {
           type="text"
           inputmode="numeric"
           pattern="[0-9]*"
-          className="component-direct-debit-form__text-field focus-target"
           label="Account number"
           error={props.accountNumberError}
         />
       </div>
 
-      <div className="component-direct-debit-form__confirmation-css-checkbox">
-        <CheckBoxWithError
+      <div css={spaceBetween}>
+        <Checkbox
           id="account-holder-confirmation"
           onChange={e => props.onChange('accountHolderConfirmation', props.updateAccountHolderConfirmation, e)}
           checked={props.accountHolderConfirmation}
-          text="I confirm that I am the account holder and I am solely able to authorise debit from
+          supporting="I confirm that I am the account holder and I am solely able to authorise debit from
           the account"
           error={props.accountHolderConfirmationError}
         />
       </div>
-
-      <Button
-        id="qa-direct-debit-submit"
-        onClick={e => props.onSubmit(e)}
-      >
+      <ThemeProvider theme={buttonReaderRevenueBrand}>
+        <Button
+          id="qa-direct-debit-submit"
+          onClick={e => props.onSubmit(e)}
```

Comment:
```suggestion
          onClick={props.onSubmit}
```

Reply:
Thanks - I hadn't understood the default event param. Good to know about this!